SU-HSDO/suhumsci
# SHS-5868: Uninstall admin login path module

# READY FOR REVIEW

## Summary
- Follow up to [this comment](https://github.com/SU-HSDO/suhumsci/pull/1401#issuecomment-2341294896), adds an update hook to properly uninstall `admin_login_path` module. Will remove the module from `composer.json` in a follow-up PR to be added to a future release.

## Need Review By (Date)
10/23

## Urgency
medium

## Steps to Test
1. Visit `/admin/modules`, search for "admin login path" and confirm that the "Admin Login Path" module is disabled.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208572384688698
